Cargo.toml

```diff
@@ -26,6 +26,9 @@ encase = "0.4.1"
 log = "0.4.17"
 rand = "0.8.5"
 
+[dev-dependencies]
+bevy = { version = "0.9.1" }
```

Comment:
I'm pretty sure that adding bevy with it's default features here is the only change that's needed.